server/routes/__tests__/index.spec.js

```diff
@@ -44,6 +50,17 @@ describe('GET /', () => {
   });
 
   describe('Homepage', () => {
+    const establishmentPersonalisationToggle = jest.fn();
+
+    const testUser = new User({
+      prisonerId: 'A1234BC',
+      firstName: 'Test',
+      surname: 'User',
+      bookingId: 1234567,
+    });
```

Comment:
Feasibly this could be mutated. Overkill, but could be moved to the `beforeEach` block.